addons/project/static/src/components/project_right_side_panel/project_right_side_panel.xml

```diff
@@ -2,18 +2,19 @@
 <templates id="template" xml:space="preserve">
 
     <t t-name="project.ProjectRightSidePanel" owl="1">
-        <div t-if="projectId" class="o_rightpanel">
-            <ProjectRightSidePanelSection
+        <div t-if="projectId" class="o_rightpanel pt-0 bg-white border-start overflow-auto">
```

Comment:
Not a big fan of tablet's layout :/
Can we do something  about it?
![image](https://user-images.githubusercontent.com/8558351/188655638-a524aaa3-84d0-4e2e-b664-2c1f011a100f.png)

Reply:
You can re-check 👍